src/appleseed/renderer/modeling/postprocessingstage/renderstamppostprocessingstage.cpp

```diff
@@ -103,7 +115,7 @@ namespace
         {
             // Render stamp settings.
             const auto Font = TextRenderer::Font::UbuntuL;
-            const float FontHeight = 14.0f;
+            const float FontHeight = 14.0f * m_scale_factor;
```

Comment:
`FontHeight` is no longer a true constant (which would be marked as `constexpr` if all our compilers would support it), thus it must be spelled `font_height`.